module_8/backend/src/services/config/configuration.ts

```diff
@@ -0,0 +1,4 @@
+export default () => ({
```

Comment:
don't really see how this file is used. Can you pls clarify a bit?

Reply:
I forget delete it. Fixed